src/core/connect-extension/connect-extension-web.web.ts

```diff
@@ -31,8 +31,8 @@ export const ConnectExtensionWeb = (() => {
         try {
             // store session
             await storeEncrypted(JSON.stringify(conn), CONN_EXTENSION, CONN_EXTENSION);
```

Comment:
a good pattern would be to return a `Promise.resolve()` otherwise the returned promise will not ever resolve